### PR TITLE
[SHELL32][USERENV] Update Turkish (tr-TR) translation

### PR DESCRIPTION
**Pull request by scropool**
## Purpose

correcting incorrect Turkish lines and translating almost all English lines into Turkish

the help of Turkish translators is very important in this PR, please let's do our best

### CODE REVIEW

**Comment by scropool** [comment]
Cc @reactos/lang-turkish, and also @Zaryob @mahirgul @vsaucing for review

File: dll/win32/userenv/lang/tr-TR.rc [hunk 1]
```diff
@@ -1,4 +1,10 @@
-/* TRANSLATOR: 2014, 2020 Erdem Ersoy (eersoy93) (erdemersoy [at] erdemersoy [dot] net) */
+/*
+ * PROJECT:     ReactOS User Environment DLL
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Turkish resource file
+ * TRANSLATORS: Copyright 2014-2020 Erdem Ersoy (eersoy93) <erdemersoy@erdemersoy.net>
```
Comment (JoachimHenze) [inline dll/win32/userenv/lang/tr-TR.rc:5]:
```suggestion
 * TRANSLATORS: Copyright 2014-2020 Erdem Ersoy (eersoy93) <erdemersoy@erdemersoy.net>
```

**Review by JoachimHenze — APPROVED** [review]
Approved.

File: dll/win32/userenv/lang/tr-TR.rc [hunk 2]
```diff
@@ -1,4 +1,10 @@
-/* TRANSLATOR: 2014, 2020 Erdem Ersoy (eersoy93) (erdemersoy [at] erdemersoy [dot] net) */
+/*
+ * PROJECT:     ReactOS User Environment DLL
```
Comment (binarymaster) [inline dll/win32/userenv/lang/tr-TR.rc:2]:
```suggestion
 * PROJECT:     ReactOS Win32 User Environment
```

File: dll/win32/userenv/lang/tr-TR.rc [hunk 3]
```diff
@@ -1,4 +1,10 @@
...
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Turkish resource file
+ * TRANSLATORS: Copyright 2014-2020 Erdem Ersoy (eersoy93) <erdemersoy@erdemersoy.net>
+ *              Copyright 2024 Ethem De Santa (scropool) <oyunyumru4@gmail.com>
```
Comment (binarymaster) [inline dll/win32/userenv/lang/tr-TR.rc:6]:
```suggestion
 *              Copyright 2021 Süleyman Poyraz <zaryob.dev@gmail.com>
 *              Copyright 2024 Ethem De Santa (scropool) <oyunyumru4@gmail.com>
```

Comment (scropool) [reply dll/win32/userenv/lang/tr-TR.rc:7]: thanks for suggestion then commiting 👍

**Comment by JoachimHenze** [comment]
Why closing that PR @scropool ? I did like the change with the Cookies folder, and some of the other places where strings were shortened!

**Pull request closed without merging**